### ISSUE TITLE
override chronological order for specified items

### ISSUE THREAD

**Issue by iKL3W** [EthanStandel/browser-rss-parser#7]
Is it possible to add a new property to RSS items like "priority" and if the property is set to true then no matter how old the article is it would just display it first, before any other article of the ul element?

**Comment by EthanStandel** [comment]
I don't think we really have a way to do that for particular _articles._ Because we kind of just get the articles from the RSS feeds. I guess we could have a running list of of "important" article titles and then when we're going through the feeds, we can sort those titles to the top. It's not foolproof, but it'll work in a good amount of cases.

**Comment by iKL3W** [comment]
Well, if this work for some cases, it's already progress!
